[2024-10-27T14:54:27.630+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-27T14:54:27.676+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag2.first_task manual__2024-10-27T14:54:26.703772+00:00 [queued]>
[2024-10-27T14:54:27.694+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag2.first_task manual__2024-10-27T14:54:26.703772+00:00 [queued]>
[2024-10-27T14:54:27.694+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-27T14:54:27.751+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2024-10-27 14:54:26.703772+00:00
[2024-10-27T14:54:27.769+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=609) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-27T14:54:27.772+0000] {standard_task_runner.py:72} INFO - Started process 613 to run task
[2024-10-27T14:54:27.774+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag2', 'first_task', 'manual__2024-10-27T14:54:26.703772+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmppz06wl75']
[2024-10-27T14:54:27.776+0000] {standard_task_runner.py:105} INFO - Job 56: Subtask first_task
[2024-10-27T14:54:27.950+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag2.first_task manual__2024-10-27T14:54:26.703772+00:00 [running]> on host 2d730dc1e71d
[2024-10-27T14:54:28.127+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-27T14:54:26.703772+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-27T14:54:26.703772+00:00'
[2024-10-27T14:54:28.128+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-27T14:54:28.165+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-27T14:54:28.166+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-10-27T14:54:28.175+0000] {subprocess.py:86} INFO - Output:
[2024-10-27T14:54:28.177+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-10-27T14:54:28.178+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-27T14:54:28.225+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-27T14:54:28.225+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag2, task_id=first_task, run_id=manual__2024-10-27T14:54:26.703772+00:00, execution_date=20241027T145426, start_date=20241027T145427, end_date=20241027T145428
[2024-10-27T14:54:28.268+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-27T14:54:28.296+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-27T14:54:28.317+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
